arrows/ocv/tests/test_crop_chips.cxx

```diff
@@ -0,0 +1,182 @@
+  EXPECT_NE(nullptr, algo) << "Unable to create crop_chips algorithm of type ocv";
+  algo::crop_chips* algo_ptr = algo.get();
+  ASSERT_TRUE(typeid(*algo_ptr) == typeid(ocv::crop_chips)) << "Factory method did not construct the correct type";
+}
+
+
+namespace {
+
+// TODO: The following functions create dummy image data for implementing
+// tests. They have been copied from `kwiver/arrows/ocv/tests/test_image.cxx`.
+// It would be useful to separate them out into a helpers file to avoid code
+// duplication.
+
+// helper function to populate the image with a pattern
+// the dynamic range is stretched between minv and maxv
+template <typename T>
+void
+populate_ocv_image(cv::Mat& img, T minv, T maxv)
+{
+  const double range = static_cast<double>(maxv) - static_cast<double>(minv);
+  const double offset = - minv;
+  const unsigned num_c = img.channels();
+  for( unsigned int p=0; p<num_c; ++p )
+  {
+    for( unsigned int j=0; j<static_cast<unsigned int>(img.rows); ++j )
+    {
+      for( unsigned int i=0; i<static_cast<unsigned int>(img.cols); ++i )
+      {
+        const double pi = 3.14159265358979323846;
+        double val = ((std::sin(pi*double(i)*(p+1)/10) * std::sin(pi*double(j)*(p+1)/10))+1) / 2;
+        img.template ptr<T>(j)[num_c * i + p] = static_cast<T>(val * range + offset);
+      }
+    }
+  }
+}
+
+
+// helper function to populate the image with a pattern
+template <typename T>
+void
+populate_ocv_image(cv::Mat& img)
+{
+  const T minv = std::numeric_limits<T>::is_integer ? std::numeric_limits<T>::min() : T(0);
+  const T maxv = std::numeric_limits<T>::is_integer ? std::numeric_limits<T>::max() : T(1);
+  populate_ocv_image(img, minv, maxv);
+}
+
+
+// helper function to populate the image with a pattern
+// the dynamic range is stretched between minv and maxv
+template <typename T>
+void
+populate_vital_image(kwiver::vital::image& img, T minv, T maxv)
+{
+  const double range = static_cast<double>(maxv) - static_cast<double>(minv);
+  const double offset = - minv;
+  for( unsigned int p=0; p<img.depth(); ++p )
+  {
+    for( unsigned int j=0; j<img.height(); ++j )
+    {
+      for( unsigned int i=0; i<img.width(); ++i )
+      {
+        const double pi = 3.14159265358979323846;
+        double val = ((std::sin(pi*double(i)*(p+1)/10) * std::sin(pi*double(j)*(p+1)/10))+1) / 2;
+        img.at<T>(i,j,p) = static_cast<T>(val * range + offset);
+      }
+    }
+  }
+}
+
+
+// helper function to populate the image with a pattern
+template <typename T>
+void
+populate_vital_image(kwiver::vital::image& img)
+{
+  const T minv = std::numeric_limits<T>::is_integer ? std::numeric_limits<T>::min() : T(0);
+  const T maxv = std::numeric_limits<T>::is_integer ? std::numeric_limits<T>::max() : T(1);
+  populate_vital_image<T>(img, minv, maxv);
+}
+
+
+} // end anonymous namespace
+
+
+TEST(crop_chips, simple)
+{
+  using namespace kwiver;
+  using namespace kwiver::arrows;
+
+  kwiver::vital::image_of<uint8_t> img(200,300,3);
```

Comment:
I should note that `kwiver::vital` is unneeded here due to earlier `using namespace`, but a) that doesn't make it harmful, and b) this is true in other tests, also, so at least we're consistent.

That said, I notice that your use of whitespace is not very consistent. In particular, please at least add space after the `,`s here. (Also, is it useful for this test to operate on more than one image type, e.g. 1-, 3- and 4-channel? Or would testing all those be overkill?)